nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/users/SuggestUserEntries.java

```diff
@@ -143,6 +143,12 @@
     @Param(name = "lang", required = false)
     protected String lang;
 
+    /*
+     * @since 11.4
+     */
+    @Param(name = "allowSubGroupsRestriction", required = false, description = "Suggest users from the subgroups specified in the group restriction.")
```

Comment:
It sounds like we're only suggesting from the subgroups, not including the restricted group itself.
Maybe something like "Include the subgroups when passing a groupRestriction."